vault/auth.go

```diff
@@ -198,7 +198,7 @@ func (v *Vault) EC2Login() string {
 			nonce = val
 		}
 		fs := vfs.OS()
-		f, err := fs.OpenFile(output, os.O_WRONLY | os.O_CREATE | os.O_TRUNC, os.FileMode(0600))
+		f, err := fs.OpenFile(output, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, os.FileMode(0600))
```

Comment:
any reason this change is part of this PR?

Reply:
Gofmt change

Reply:
ok, if you rebase I think this'll go away - this was done in #302